### ISSUE TITLE
Support escape and tab shortcuts to move out

### ISSUE THREAD

**Issue by KannarFr** [anubra266/choc-autocomplete#152]
When autocomplete is focused, the escape & tab shortcuts are not handled.

**Comment by sukhbains** [comment]
I agree this is needed for accessibility.  Tab should go to the next focusable elements. If there is a list that should be selectable through the keyboard arrows.
